src/exec/Shelley.hs

```diff
@@ -106,4 +108,5 @@ mkParams rr cdb nconf nstate blockchainTime = NodeArgs
   , blockMatchesHeader = nodeBlockMatchesHeader
   , maxUnackTxs = maxBound
   , chainSyncPipelining = pipelineDecisionLowHighMark 200 300 -- TODO: make configurable!
+  , mempoolCap = MempoolCapacity 2048 -- TODO: is this value ok?
```

Comment:
Suggestions for a proper cap?

Reply:
It should probably be the same order of magnitude as the configured cardano-sl limit, which I think is rather lower.

`lib/configuration.yaml`
```
    memPoolLimitTx: 200 # mem pool will be limited to this many transactions
```
So lets go with that.

Reply:
Done